### PR TITLE
Add input ids for labels

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/oQEjV78v/650-2-a11y-forms-input-ids-for-labels)

### CODE REVIEW

**Comment by HSalmon** [comment]
@robguthrie, @rdbartlett

**Comment by gdpelican** [comment]
What's the impetus behind doing this?

**Comment by HSalmon** [comment]
@gdpelican - accessibility (for screen readers)

**Comment by robguthrie** [comment]
This means the label is associated with the form field it's labelling. When you click the label it'll focus the field and screen readers can read the label out when the field is focused. It's pretty great.

**Comment by gdpelican** [comment]
:+1:

File: lineman/app/components/contact_page/contact_page.haml [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
-    %input#contact-name-field.form-control{ng-model: 'message.name', ng_required: 'true', placeholder: '{{contact_message_form.name_placeholder | translate}}'}
+    %input.form-control#contact-name-field{ng-model: 'message.name', ng_required: 'true', placeholder: '{{contact_message_form.name_placeholder | translate}}'}
 
     %label{for: 'contact-email-field', translate: 'contact_message_form.email_label'}
-    %input#contact-email-field.form-control{ng-model: 'message.email', ng_required: 'true', placeholder: '{{contact_message_form.email_placeholder | translate}}'}      
+    %input.form-control#contact-email-field{ng-model: 'message.email', ng_required: 'true', placeholder: '{{contact_message_form.email_placeholder | translate}}'}
 
     %label{for: 'contact-message-field', translate: 'contact_message_form.message_label'}
-    %textarea#group-description-field.form-control{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
+    %textarea#contact-message-field{ng-model: 'message.message', placeholder: '{{contact_message_form.message_placeholder | translate}}'}
```
Comment (gdpelican) [inline lineman/app/components/contact_page/contact_page.haml]: Why remove form-control here? And do we use this contact page at all?

Comment (HSalmon) [inline lineman/app/components/contact_page/contact_page.haml]: I didn't mean to remove it, but no, I don't think this page is used. @robguthrie, should we remove the contact stuff for now?

Comment (robguthrie) [inline lineman/app/components/contact_page/contact_page.haml]: is it this page? https://www.loomio.org/pages/contact

Comment (HSalmon) [inline lineman/app/components/contact_page/contact_page.haml]: That link doesn't work for me in angular

Comment (robguthrie) [inline lineman/app/components/contact_page/contact_page.haml]: it's a public page .. try with a private browser. it's linked from the
homepage footer

**Comment by robguthrie** [comment]
@HSalmon just FYI I've merged this and we'll probably keep a note of these changes for an update to the a11y group

**Pull request merged**